hkmc2/shared/src/main/scala/hkmc2/bbml/bbML.scala

```diff
@@ -442,15 +442,15 @@ class BBTyper(using elState: Elaborator.State, tl: TL):
             effBuff += eff
             ctx += sym -> rhsTy
             goStats(stats)
-          case TermDefinition(_, Fun, sym, ps :: Nil, _, sig, S(body), _, _, _) :: stats =>
+          case TermDefinition(_, Fun, sym, ps :: Nil, _, sig, S(body), _, _, _, _) :: stats =>
```

Comment:
Instead of adding more meaningless underscores, better leave this place in a better state than you found it by using this recent Scala 3 feature:
```suggestion
          case td @ TermDefinition(k = Fun, params = ps :: Nil, body = S(body)) :: stats =>
```
(and use `td.sym` instead of `sym`)